calm-visualizer/src/components/graph/Graph.tsx

```diff
@@ -12,84 +12,128 @@ interface Props {
 
 // const groups: string[] = [];
 
-function createConnectsRelationship(instance: BrowserJsPlumbInstance, relationship: CALMConnectsRelationship) {
-    const r = relationship["relationship-type"]["connects"];
-    instance.connect({
-        source: document.getElementById(r.source.node)!,
-        target: document.getElementById(r.destination.node)!,
-        anchor: "Continuous",
-        connector: {
-            type:"Straight",
-            options: {
-                "stub": 25
+function createConnectsRelationship(instance: BrowserJsPlumbInstance, relationships: CALMConnectsRelationship[]) {
+    relationships.map(
+            relationship =>{
+                const connects  = relationship["relationship-type"]["connects"];
+                instance.connect({
+                    source: document.getElementById(connects.source.node)!,
```

Comment:
Seeing `getElementById` in a React site gives me pause. I'm not a React dev but is there a better way to look this up?
@aidanm3341  might be good to get your thoughts here?

Reply:
this is my bad for doing it the quick and dirty way on the hackday. There is a better way using react ref's instead, but I don't think it should be a blocker for this PR. It would be good for us to refactor this before merging into main though